backend/custom_models/sample_emib.json

```diff
@@ -0,0 +1,39 @@
+[
```

Comment:
Why do we need this file? For testing?

Reply:
shoot.... didn't mean to push that one.......

Reply:
Dropped in d3cc59c